recipes/magnum/all/patches/2020.06/0001-emscripten-toolchain.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 19e6c2d..feda9a4 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -276,10 +276,6 @@ if(TARGET_VK)
+     set(MAGNUM_TARGET_VK 1)
+ endif()
+ 
+-if(CORRADE_TARGET_EMSCRIPTEN)
```

Comment:
why not needed? what does `UseEmscripten` do?

Reply:
The source of the patch is documented, and the reasons are explained by the library author himself: https://github.com/mosra/magnum/issues/490. Basically he was providing his own Emscripten toolchain in the past because the _official_ ones were quite unstable. New versions already remove this custom toolchain.